esports_data_from_rimble/Makefile

```diff
@@ -40,29 +40,36 @@ build: tmp/x.wasm
 .PHONY: match-winner
 match-winner: check build
 	@echo "Running WASM module..."
-	@sed -e 's/{{ .YOUR_RIMBLE_API_KEY }}/$(RIMBLE_API_KEY)/' ./match-winner.json.template > ./tmp/match-winner.json
+	@sed -e 's/{{ .YOUR_RIMBLE_MATCH_DATE }}/$(RIMBLE_MATCH_DATE)/' \
+	     -e 's/{{ .YOUR_RIMBLE_MATCH_ID }}/$(RIMBLE_MATCH_ID)/' \
+	     -e 's/{{ .YOUR_RIMBLE_API_KEY }}/$(RIMBLE_API_KEY)/' \
+	     ./match-winner.json.template > ./tmp/match-winner.json
```

Comment:
Previously, I'd tried updating the example code to fetch a matchid. Per your feedback, however, we expect the user to pass in the match id and date. This allows us to keep the actual example code (i.e., main.go) largely unchanged.